codebase2/codebase-sqlite/U/Codebase/Sqlite/Reference.hs

```diff
@@ -4,15 +4,17 @@
 
 module U.Codebase.Sqlite.Reference where
 
-import Control.Applicative (liftA3)
 import Data.Tuple.Only (Only (..))
 import U.Codebase.Reference (Id' (Id), Reference' (ReferenceBuiltin, ReferenceDerived))
 import U.Codebase.Sqlite.DbId (HashId, ObjectId, TextId)
 import U.Codebase.Sqlite.LocalIds (LocalDefnId, LocalHashId, LocalTextId)
+import Unison.Prelude
 import Unison.Sqlite (FromField, FromRow (fromRow), RowParser, SQLData (SQLNull), ToField, ToRow (toRow), field)
 
 type Reference = Reference' TextId ObjectId
 
+type TextReference = Reference' Text Text
```

Comment:
Can we make this `Reference' Text Base32Hex` instead, or does that do something terrible dependencies-wise?